rdflib/plugins/serializers/trig.py

```diff
@@ -83,7 +68,10 @@ def serialize(self, stream, base=None, encoding=None,
             if self.default_context and store.identifier==self.default_context:
                 self.write(self.indent() + '\n{')
             else:
-                self.write(self.indent() + '\n<%s> {' % self.getQName(store.identifier))
+                iri = self.getQName(store.identifier)
+                if iri is None:
+                    iri = '<%s>' % store.identifier
+                self.write(self.indent() + '\n%s {' % iri)
```

Comment:
two questions here:
 - should it be possible that `store.identifier` isn't in `getQName` at all?
 - wouldn't this introduce a bug if `iri is not None` cause it's not wrapped in `<%s>` anymore?
    (so shouldn't line 73 be: `iri = '%s' % store.identifier` and 74: `self.write(self.indent() + '\n<%s> {' % iri)` ?